assets/app/scripts/controllers/images.js

```diff
@@ -23,10 +24,37 @@ angular.module('openshiftConsole')
       LabelFilter.setLabelSuggestions($scope.labelSuggestions);
       $scope.imageStreams = LabelFilter.getLabelSelector().select($scope.unfilteredImageStreams);
       $scope.emptyMessage = "No image streams to show";
+      updateMissingStatusTags();
       updateFilterWarning();
       Logger.log("image streams (subscribe)", $scope.imageStreams);
     }));
 
+    // Check each image stream to see if the spec tags resolve to status tags.
```

Comment:
isn't this backwards? I thought we cared about image streams with status tags that weren't resolved to spec tags